src/components/admin/settings/general/Update.vue

```diff
@@ -53,113 +69,106 @@ const isSubmitting = ref(false);
 
 const onSubmit = handleSubmit(async values => {
   try {
+    // Validate and prepare featured links
+    const links = featuredLinks.value.map(link => ({
+      ...link,
+      title: link.title || link.url,
+    }));
+    const result = z.array(LinkSchema).safeParse(links);
+    if (!result.success) {
+      alert('Invalid links!');
+      return;
+    }
+
     isSubmitting.value = true;
     await $fetch('/api/settings/general', {
       method: 'PUT',
       body: {
         ...values,
-      }
+        organization: {
+          ...values.organization,
+          links: result.data,
+        },
+      },
     });
     emits('saved');
   } catch (error) {
-    console.error("error saving settings", error);
+    console.error('error saving settings', error);
   } finally {
     isSubmitting.value = false;
   }
 });
 </script>
+
 <template>
   <div class="">
     <!-- Organization Settings -->
-    <div class="px-4 space-y-6 w-full items-center mt-4"  v-if="forms == 'general' || forms == 'all'">
-      <section class="w-full md:w-1/3" v-if="forms == 'all'">
-        <h2 class="text-base font-bold text-zinc-900 font-noto">Organization Settings</h2>
-        <h4 class="text-sm mb-5 text-zinc-400">Configure career site's home page.</h4>
-      </section>
-      <section class="w-full md:w-2/3">
-        <div class="flex items-end">
-          <div class="mr-4">
-            <img class="w-16 h-16 md:w-20 md:h-20 rounded-xl" :src="url"
-              width="80" height="80" alt="User upload" />
+    <div class="px-4 space-y-6 w-full">
+      <section>
+        <h4 class="text-lg font-medium text-gray-900">Organization</h4>
+        <div class="md:flex gap-4 items-center">
+          <div class="w-full md:w-2/3">
+            <label class="block text-sm font-medium mb-1 text-zinc-900 font-noto" for="name">Organization Name</label>
+            <input class="input-custom" type="text" v-model="organizationName" />
```

Comment:
I don't know what's the case, but you are undoing my changes. update your fork then push.